Sources/PostgresNIO/New/PSQLError.swift

```diff
@@ -180,6 +180,15 @@ struct PostgresCastingError: Error, Equatable {
     }
 }
 
+extension PostgresDecodingError: CustomStringConvertible {
+    var description: String {
```

Comment:
This property must now be marked `public`.

We even do have a test case for it. However the test case has one big bug, that we should fix now (in `PostgresErrorTests.swift`):

```swift
    func testPostgresCastingErrorDescription() {
        let error = PostgresCastingError(
            code: .typeMismatch,
            columnName: "column",
            columnIndex: 0,
            targetType: String.self,
            postgresType: .text,
            postgresFormat: .binary,
            postgresData: ByteBuffer(string: "hello world"),
            file: "foo.swift",
            line: 123
        )

        XCTAssertNotEqual("\(error)", "Database error") // <-- Remove the Not 🤦‍♂️
    }
```

FYI: @gwynne I guess bug of a century.